ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -695,6 +700,49 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
```

Comment:
Are these points necessary to bring in? We should be able to tell from the command and status if the valve is in manual override.